src/ompl/tools/thunder/Thunder.h

```diff
@@ -110,6 +110,7 @@ namespace ompl
             double DenseD_{};
             double SparseD_{};
             size_t n_threads_{0};
+            size_t cforest_n_threads_{0};
```

Comment:
what is the benefit of this differing from `n_threads_` above? would it be too confusing to use the same n_threads_ we define / set already in RRTConnect and CForest depending on what we're using? right now it's unused in cforest planning. 
```
 if (planner_type_ == thunderPlanner::PLANNER_CFOREST) {
          std::vector<base::PlannerPtr> planner_vec {1};
          planner_vec_ = planner_vec;
        } else {
          std::vector<base::PlannerPtr> planner_vec (n_threads_);
          planner_vec_ = planner_vec;
        }
```

Reply:
if it's more clear / beneficial to have them as separate variables maybe we should rename `n_threads_` to `rrt_connect_n_threads`?

Reply:
As things are, it will be okay to use n_threads_ perhaps.
But really what n_threads_ is specifying is how many planners are used in parallel plan. The planner themselves are able to use more than one thread, and this is the case for CForest.
What we can do is, if we find it better, send the same value to both from dig. But in ompl they mean different things, so I find they should be seperate.